src/Faker/Provider/it_IT/Payment.php

```diff
@@ -16,4 +16,45 @@ public static function bankAccountNumber($prefix = '', $countryCode = 'IT', $len
     {
         return static::iban($countryCode, $prefix, $length);
     }
+
+    /**
+     * Source:
+     * @see https://en.wikipedia.org/wiki/List_of_banks_in_Italy
+     */
+    protected static $banks = array(
+        'UniCredit',
```

Comment:
Could you sort this list alphabetically and group bank names by capital letter? This will help tracking future additions / deletions.